test/performance/odp_crypto.c

```diff
@@ -103,7 +103,7 @@ typedef struct {
 	int in_place;
 
 	/**
-	 * If non zeor output of previous operation taken as input for
+	 * If non zero output of previous operation taken as input for
```

Comment:
`non-zero` would be better and comma could be added. E.g. `If non-zero, output...`